### PR TITLE
Math Magicians Components

### PR DESCRIPTION
**Pull request by Munsa1**
# Components

### In this milestone🧹:

- I have deleted all the boilerplate from CRA (text, images, styles).
- I have created a directory called components.
- I have Inside components, created a new Calculator.js file.
- I have developed just the UI of the calculator; the math logic to make it work will be added in the next project.
- I have used class-based components.

Live Demo link (http://localhost:3000)

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+// eslint-disable-next-line
+import CalculatorBtn from './CalculatorBtn';
```
Comment (dasileker) [inline src/components/Calculator.js:3]: - [ ] Please fix the typo error  `'./CalculatorBtn';` should be `'./calculatorBtn';`.

Capture web_23-11-2021_131837_localhost

- After update 

Capture web_23-11-2021_131910_localhost

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Munsa1 ,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Munsa1) [reply src/components/Calculator.js:3]: Thanks for the feedback but my file is Capital not a small letter, changing it is giving me an error here

Comment (Munsa1) [reply src/components/Calculator.js:3]: The issue has been resolved

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**